src/main/java/fi/dy/masa/litematica/util/ItemUtils.java

```diff
@@ -128,7 +128,7 @@ private static void overrideStackSize(BlockState state, ItemStack stack)
 
     public static ItemStack storeTEInStack(ItemStack stack, BlockEntity te)
     {
-        NbtCompound nbt = te.createNbtWithId();
+        NbtCompound nbt = te.createNbtWithId(null);
```

Comment:
Trying to get a RegistryWrapper. I found this would work but not sure if this is a good idea:
```
MinecraftClient.getInstance().world.getRegistryManager()
```

![Uploading Screenshot 2024-03-10 at 9.14.54 PM.png…]()

Reply:
I don't think most classes use the registryLookup but this could crash the game by passing in null if a class would use it. Each Block Entity can have their own writeNbt function that might use this